bindings/java/README.md

```diff
@@ -62,16 +62,48 @@ You can use Maven to build both Rust dynamic lib and JAR files with one command
 ./mvnw clean package -DskipTests=true
 ```
 
+## Setup Tests
+
+Please copy `.env.example` to `src/test/resources/.env` and change the values on need.
```

Comment:
So we can read `.env` directly like `dotenvy` does?

Reply:
Yes, I used `Properties` to read the `.env` file and then used `System.getenv()` to read the environment. If variables with the same names already exist in the environment, then their values will be preserved.

Reply:
I'm talking about remove `.env` at `/.env` instead of `bindings/java/src/test/resources/.env` so we can reuse the same env file across the whole project.

Reply:
Okay, I'll try.